spec/sass/basic/input.scss

```diff
@@ -1 +0,0 @@
-@import "input.sass";
```

Comment:
I read the commit message. Please don't change this. It tests an important feature of Sass, being able to mix syntaxes in a single run.